File: src/main/java/com/devspace/conexfy/converters/ConHttpMethodEnumToStringWritingConverter.java
package com.devspace.conexfy.converters;

import org.springframework.core.convert.converter.Converter;
import org.springframework.data.convert.WritingConverter;
import org.springframework.lang.NonNull;

import com.devspace.conexfy.enums.ConHttpMethodEnum;

@WritingConverter
public class ConHttpMethodEnumToStringWritingConverter implements Converter<ConHttpMethodEnum, String> {

    @Override
    public String convert(@NonNull ConHttpMethodEnum source) {
        return source.name();
    }
}

File: src/main/kotlin/com/devspace/conexfy/converters/ConHttpMethodEnumToStringWritingConverter.kt
package com.devspace.conexfy.converters

import com.devspace.conexfy.enums.ConHttpMethodEnum
import org.springframework.core.convert.converter.Converter
import org.springframework.data.convert.WritingConverter
import org.springframework.lang.NonNull

@WritingConverter
class ConHttpMethodEnumToStringWritingConverter : Converter<ConHttpMethodEnum, String> {
    override fun convert(@NonNull source: ConHttpMethodEnum): String {
        return source.name
    }
}
